src/workerd/api/global-scope.h

```diff
@@ -442,7 +442,8 @@ class ServiceWorkerGlobalScope: public WorkerGlobalScope {
       kj::HttpService::Response& response,
       kj::Maybe<kj::StringPtr> cfBlobJson,
       Worker::Lock& lock,
-      kj::Maybe<ExportedHandler&> exportedHandler);
+      kj::Maybe<ExportedHandler&> exportedHandler,
+      kj::Maybe<jsg::Ref<AbortSignal>> abortSignal = kj::none);
```

Comment:
I think the call site in worker-entrypoint is the _only_ call site for this method? In that case, there's no need to supply a default for this new argument, and in fact it doesn't even need to be `Maybe`.

If there are other call sites, it's probably worth updating them to pass `kj::none` explicitly -- this provides a chance to review whether maybe one or more of them should actually be passing an AbortSignal.

Reply:
There was one other call site (TestFixture::runRequest). I also made the parameter mandatory in the Request class too which had only a handful of callsites too.